fintech-examples/fintech-ui/src/app/bank/list-accounts/list-accounts.component.html

```diff
@@ -1,6 +1,7 @@
 <div>
   <div class="lacc-parent">
     <div
+      class="withoutOutline"
```

Comment:
As far as I understand we wanted the outline not to be removed but wrap around the element without the extra padding on the right

Reply:
We don't need the browser outline anymore because we apply a card shadow there and I think it should be fine